ansible/roles/postgresql/templates/pgbouncer.ini.j2

```diff
@@ -0,0 +1,207 @@
+;; database name = connect string
+;;
+;; connect string params:
+;;   dbname= host= port= user= password=
+;;   client_encoding= datestyle= timezone=
+;;   pool_size= connect_query=
+[databases]
+{% for db in postgresql_dbs | unique %}
+{{ db }} = host=localhost port=5432
```

Comment:
By that I mean, does this port overlap with postgres and do we need to change what port postgres listens on?